tests/core/test_accounts.py

```diff
@@ -57,3 +63,59 @@ def test_password_change(self):
                 password=example_data['password1']
             )
         )
+
+
+class UpdateProfileViewTest(RequireRootArticleMixin, ArticleWebTestUtils, DjangoClientTestBase):
+
+    def test_update_profile(self):
+        self.client.post(
+            resolve_url('wiki:profile_update'),
+            {"email": "test@test.com", "password1": "newPass", "password2": "newPass"},
+            follow=True
+        )
+
+        test_auth = authenticate(username='admin', password='newPass')
+
+        self.assertNotEqual(test_auth, None)
+        self.assertEqual(test_auth.email, 'test@test.com')
+
+
+@wiki_override_settings(ACCOUNT_HANDLING=True)
+class LogoutViewTests(RequireRootArticleMixin, DjangoClientTestBase):
+
+    def test_logout_account_handling(self):
+        self.client.get(wiki_settings.LOGOUT_URL)
+        user = auth.get_user(self.client)
+        self.assertFalse(auth.get_user(self.client).is_authenticated)
+        self.assertIsInstance(user, AnonymousUser)
+
+
+@wiki_override_settings(ACCOUNT_HANDLING=True)
+class LoginTestViews(RequireRootArticleMixin, TestBase):
+    def test_already_signed_in(self):
+        self.client.force_login(self.superuser1)
+        response = self.client.get(wiki_settings.LOGIN_URL)
+        self.assertRedirects(response, reverse('wiki:root'))
+
+    def test_log_in(self):
+        self.client.post(
+            wiki_settings.LOGIN_URL,
+            {'username': SUPERUSER1_USERNAME, 'password': SUPERUSER1_PASSWORD}
+        )
+        self.assertTrue(self.superuser1.is_authenticated, True)
```

Comment:
This should have been `self.assertIs`.

Reply:
This it cannot be... because it's either a boolean `True` (Django 2.0) or a `CallableBoolean(True)` (Django 1.11)

Reply:
Sorry should have added a comment, is done now